bigquery_etl/glam/clients_daily_histogram_aggregates.py

```diff
@@ -111,11 +111,6 @@ def main():
         + (" --no-parameterize" if args.no_parameterize else "")
     )
 
-    # This enables build_id filtering against Buildhub data
-    # and filtering out of an erroneous version "1024" (see query template for details).
-    # Only the desktop builds are reported to Buildhub
-    filter_desktop_builds = True if args.product == "firefox_desktop" else False
```

Comment:
Why don't we start by just setting `filter_desktop_builds` to `False`?

Reply:
We can start with that to be safe, but I expect this to be fully removed at some point, as we're not using this filter anywhere, and leaving this here can be confusing in the long run.